main/modes/picross/picross_menu.c

```diff
@@ -67,13 +73,17 @@ static const char str_exit[] = "Exit";
 static const char str_options[] = "Options";
 
 static const char str_back[] = "Back";
-static const char str_HintsOn[] = "Hints: On";
-static const char str_HintsOff[] = "Hints: Off";
+static const char str_HintsOn[] = "Mistake Alert: On";
+static const char str_HintsOff[] = "Mistake Alert: Off";
+static const char str_GuidesOn[] = "Guides: On";
+static const char str_GuidesOff[] = "Guides: Off";
+static const char str_AnimateBGOn[] = "BG Animate: On";
+static const char str_AnimateBGOff[] = "BG Animate: Off";
 static const char str_eraseProgress[] = "Erase Progress";
 
 swadgeMode modePicross =
 {
-    .modeName = "pi-Cross",//set here and also above
+    .modeName = "pi-cross",//set name here for how name appears in "games" menu. Should match above.
```

Comment:
i think you can have it be `.modeName = str_picrossTitle,` in order to guarantee a match